internal/keycloak/groups.go

```diff
@@ -42,6 +38,7 @@ func (c *Client) rawGroups(ctx context.Context, first int) ([]byte, error) {
 		return nil, fmt.Errorf("couldn't construct groups request: %v", err)
 	}
 	q := req.URL.Query()
+	q.Add("subGroupsCount", "false")
```

Comment:
What is the reason for this change?

Reply:
This is the change that actually addresses the performance issue with groups queries, the subGroupsCount returns an integer of how many subgroups the group has, which isn't used by anything in the ssh-portal as far as I can tell